exampleCourse/questions/templateRandomNumberInput/question.html

```diff
@@ -0,0 +1,24 @@
+<pl-question-panel>
+  <p>The comparison between the correct and submitted answers can be done using
```

Comment:
Do we want to add some explanation about when you should use which method? What do you normally use?

Reply:
I think that is so personal! I avoid using sigfig. I think many students find it confusing. They think you are asking them to give that exact number of sigfigs, and if they don't know how to do it correctly, they get it marked wrong for something silly. I think tolerance is clearer to students.

Reply:
Ok, sounds good, let's go with it how it is.